sdk/eventhub/event-hubs/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 - Updates the methods on the `CheckpointStore` interface to accept
   an optional `options` parameter that can be used to pass in an
   `abortSignal` and `tracingOptions`.
+- [Breaking] Tracing options for `EventDataBatch.tryAdd` now match the shape of `OperationOptions`.
```

Comment:
Maybe put this under a `### Tracing Updates` header like we have for 5.3.0. Otherwise this looks like we should do a major version bump, but really since we treat tracing as preview, it _should_ be ok to do a minor bump. (Now I wonder if we should add a note that users depending on tracing should pin to a minor version rather than major version)

Reply:
I obey the local law of the land. Will change.

Reply:
Fixed. Also did the same for Service Bus as well.